pkg/gadgets/seccomp/gadget.go

```diff
@@ -41,6 +41,11 @@ type Trace struct {
 	client   client.Client
 
 	started bool
+
+	// policyAtTermination is used to know if there was a policy generated
+	// at pod termination so that the Generate() operation does not have
+	// to notify that it did not find a pod that matches the filter.
+	policyAtTermination bool
```

Comment:
What about calling it `policyGenerated`? Or something that indicates what this variable is for: To indicate that a policy was generated and we don't need to throw and error.

Reply:
Absolutely! I prefer `policyGenerated`.